### PR TITLE
Individual Mode

### PR DESCRIPTION
**Pull request by Bentechy66**
have a gander at this lads, should work

### CODE REVIEW

File: src/authentication/basic_auth.py [hunk 1]
```diff
@@ -54,6 +56,13 @@ def register_user(self, username, email, password, invite, **kwargs):
         token = user.email_token
         if not settings.MAIL["SEND"]:
             user.email_verified = True
+
+        if not config.get("enable_teams"):
+            team = Team(name=username, password=secrets.token_hex(32), owner=user)
+            user.save()
+            team.save()
+            user.team = team
```
Comment (AnonGuy) [inline src/authentication/basic_auth.py:64]:
```suggestion
            user.team = Team.objects.create(name=username, password=secrets.token_hex(32), owner=user)
```

**Review by AnonGuy — CHANGES_REQUESTED** [review]

**Review by AnonGuy — APPROVED** [review]

File: src/authentication/basic_auth.py [hunk 2]
```diff
@@ -54,6 +56,13 @@ def register_user(self, username, email, password, invite, **kwargs):
         token = user.email_token
         if not settings.MAIL["SEND"]:
             user.email_verified = True
+
+        if not config.get("enable_teams"):
+            team = Team(name=username, password=secrets.token_hex(32), owner=user)
```
Comment (david-cooke) [inline src/authentication/basic_auth.py]: I feel like we should make secrets.token_hex(32) the default value for the password field, but thats a discussion for another day

File: src/member/serializers.py [hunk 3]
```diff
@@ -72,6 +73,12 @@ def validate_email(self, value):
         self.instance.save()
         return value
 
+    def update(self, instance, validated_data):
+        if not config.get("enable_teams"):
+            if instance.team:
```
Comment (david-cooke) [inline src/member/serializers.py:78]: maybe merge these if statements into one?

**Review by david-cooke — APPROVED** [review]
I'm happy to merge this

**Pull request merged**